apis/projectcontour/v1/httpproxy.go

```diff
@@ -120,6 +120,12 @@ type TLS struct {
 	// backing cluster.
 	// +optional
 	Passthrough bool `json:"passthrough,omitempty"`
+	// ClientValidation defines how to verify the client certificate. This setting:
+	// 1. Enables TLS client certificate validation.
+	// 2. Requires clients to present a TLS certificate (i.e. not optional validation).
+	// 3. Specifies how the client certificate will be validated.
+	// +optional
+	ClientValidation *DownstreamValidation `json:"clientValidation,omitempty"`
```

Comment:
I wonder if there's a better word to describe what "client" is. I think it's probably best, but it gets confusing with upstream/downstream/client, etc.

Reply:
Sometimes we have used "external" to highlight that it is the client outside of the cluster. 
What about something like this:

```diff
-       // ClientValidation defines how to verify the client certificate. This setting:
+       // ClientValidation defines how to verify the client certificate when an external client establishes TLS connection to Envoy. This setting:
```

Reply:
I think the proposed replacement is an improvements 👍